src/robot_accounts.py

```diff
@@ -73,27 +72,24 @@ async def sync_robot_accounts(client, path):
             ]
             short_robot_name = target_robot.name
             target_robot.name = full_robot_name
-            print(f'- Syncing robot "{target_robot.name}".')
+            logger.info("Syncing robot", extra={"robot": target_robot.name})
             await client.update_robot(robot_id=robot_id, robot=target_robot)
-            await set_robot_secret(client, short_robot_name, robot_id)
+            await set_robot_secret(client, short_robot_name, robot_id, logger)
         # Create new robot
         else:
-            print(
-                "- Creating new robot"
-                f' "{full_robot_name}"'
+            logger.info(
+                "Creating new robot",
+                extra={"robot": target_robot.name}
             )
             try:
                 created_robot = await client.create_robot(robot=target_robot)
                 await set_robot_secret(
-                    client, target_robot.name, created_robot.id
+                    client, target_robot.name, created_robot.id, logger
                 )
             except Conflict as e:
-                print(
-                    f'''  => "{full_robot_name}"
-                    Harbor Conflict Error: {e}'''
-                )
+                logger.info("Harbor Conflict", extra={"error": e})
```

Comment:
Maybe make log level `warn` or `error`, depending how unexpected or critical this is?